containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,23 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
+    for markdown_file in $(echo "\
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Development \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Roadmap \
+      Home"
+    ); do
+      pandoc -t html -f markdown -o "${markdown_file}.html" ".wiki/${markdown_file}.md"
+    done
+    rm -rf .wiki
+
+    echo "Installing Extension Docs..."
+    rm -rf $DAFFODIL_VSCODE_DOCS_DIR
+    mkdir -p $DAFFODIL_VSCODE_DOCS_DIR
+    cp -R *.html $DAFFODIL_VSCODE_DOCS_DIR
```

Comment:
Would it be possible to generate the *.html files in a different directory, or maybe in the .wiki directocy? I could imagine someday we might add a an .html file to the root of the daffodil-vscode repo and it shouldn't be copied to the docs directory.